Technico/appsettings.json

```diff
@@ -0,0 +1,12 @@
+{
+    "ApplicationSettings": {
+        "JWT_Secret": "plaisio_ninjas_waka_maka_phone_25"
```

Comment:
The JWT_Secret is stored directly in the configuration file, which poses a security risk